### PR TITLE
NOISSUE - Add IsChannelOwner grpc endpoint

### PR DESCRIPTION
**Pull request by darkodraskovic**
Signed-off-by: Darko Draskovic <darko.draskovic@gmail.com>

Add grpc support to things service to check if a given channel belongs to a given user.

### CODE REVIEW

**Review by mteodor — DISMISSED** [review]

File: things/service_test.go [hunk 1]
```diff
@@ -1232,6 +1232,44 @@ func TestCanAccessByID(t *testing.T) {
...
+
+	chs, err := svc.CreateChannels(context.Background(), token, channel)
+	require.Nil(t, err, fmt.Sprintf("unexpected error: %s\n", err))
+	ownedCh := chs[0]
+	chs, err = svc.CreateChannels(context.Background(), token2, channel)
+	require.Nil(t, err, fmt.Sprintf("unexpected error: %s\n", err))
+	nonOwnedCh := chs[0]
+
+	_ = ownedCh
+	_ = nonOwnedCh
```
Comment (manuio) [inline things/service_test.go]: Do we need this?

File: things/service_test.go [hunk 2]
```diff
@@ -1232,6 +1232,44 @@ func TestCanAccessByID(t *testing.T) {
 	}
 }
 
+func TestIsChannelOwner(t *testing.T) {
+	token2 := "1234"
```
Comment (manuio) [inline things/service_test.go]: Maybe you can define a const next to `token`

**Review by manuio — CHANGES_REQUESTED** [review]

**Review by manuio — APPROVED** [review]
LGTM

**Review by mteodor — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**